tauv_common/config/TAM.yaml

```diff
@@ -0,0 +1,13 @@
+tam:
```

Comment:
See below. This file and thruster_manager.yaml should be moved into a subdirectory to indicate what vehicle they're describing. Maybe create a "Models" or "Vehicles" folder in the root of TAUV_common for this.